src/qiskit_qec/utils/stim_tools.py

```diff
@@ -0,0 +1,347 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name, disable=no-name-in-module
+
+"""Tools to use functionality from Stim."""
+from typing import List, Dict
+from math import log
+from stim import Circuit as StimCircuit
+from stim import DetectorErrorModel as StimDetectorErrorModel
+from stim import DemInstruction as StimDemInstruction
+from stim import DemTarget as StimDemTarget
+
+import numpy as np
+import rustworkx as rx
+
+
+from qiskit import QuantumCircuit
+from qiskit_aer.noise.errors.quantum_error import QuantumChannelInstruction
+from qiskit_aer.noise import pauli_error
+from qiskit_qec.utils.decoding_graph_attributes import DecodingGraphNode, DecodingGraphEdge
+
+
+def get_stim_circuits(circuit_dict):
+    """Converts compatible qiskit circuits to stim circuits.
+
+    Args:
+        circuit_dict: Dictionary with Qiskit circuits as values. Compatible gates are paulis,
+        controlled paulis, h, s, and sdg, swap, reset, measure and barrier. Compatible noise
+        operators correspond to a single or two qubit pauli channel.
+
+    Returns:
+        stim_circuits: TODO
+        stim_measurement_data: TODO
+    """
+    stim_circuits = {}
+    stim_measurement_data = {}
+    for circ_label, circuit in circuit_dict.items():
+        stim_circuit = StimCircuit()
+
+        # Dictionaries are not complete. For the stim definitions see:
```

Comment:
Can you put this comment information in the docstring for this method? Can you also make a new issue for it if we need to add more of them in the future?

Reply:
Done. We will need to think over whether or not the additional stim gates have a qiskit counterpart.